src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -143,6 +147,16 @@ namespace {
         if (DeckSection::hasSCHEDULE(deck))
             iterateScheduleSection( python, deck.getInputPath(), parseContext, errors, SCHEDULESection( deck ), grid, fp);
     }
+    catch (const OpmInputError& opm_error) {
+        throw;
+    }
+    catch (const std::exception& std_error) {
+        OpmLog::error(fmt::format("An error occured while loading the dynamic properties\n",
```

Comment:
The word loading is a little misleading here I think, hinting at I/O or perhaps parsing trouble. What about "while creating the dynamic simulator schedule"?